business/services.go

```diff
@@ -184,6 +198,43 @@ func (in *SvcService) GetService(namespace, service, interval string, queryTime
 		drUpdate, drDelete = getUpdateDeletePermissions(in.k8s, namespace, DestinationRules, "")
 	}()
 
+	var eTraces int
+	go func() {
```

Comment:
I like to have inline coroutines, but as this is a little bit long, I suggest if this can be done in a helper method more easy to refactor and maintain.